windows/ReactTestApp/ReactInstance.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+namespace winrt::ReactTestApp::implementation
+{
+    enum class JSBundleSource {
+        DevServer,
+        Embedded,
+    };
+
+    class ReactInstance
+    {
+    public:
+        winrt::Microsoft::ReactNative::ReactNativeHost ReactHost();
```

Comment:
We should probably return a reference and not a copy:
```suggestion
        winrt::Microsoft::ReactNative::ReactNativeHost &ReactHost();
```